src/App.jsx

```diff
@@ -26,47 +26,58 @@ import { ScholarshipsProvider } from './models/ScholarshipsContext';
-            }
-          />
-          <AuthProvider>
-            <ScholarshipsProvider>
-              <Router>
-                <Header />
-                <HeaderSkeleton />
-                <Routes>
-                  <Route
-                    path="/scholarships/new"
-                    element={<ProtectedRoute element={<AddScholarship />} />}
-                  />
-                  <Route
-                    path="/scholarships/:id/edit"
-                    element={<ProtectedRoute element={<EditScholarship />} />}
-                  />
-                  <Route
-                    path="/scholarships/:id"
-                    element={<ViewScholarship />}
-                  />
-                  <Route path="/scholarships" element={<ListScholarships />} />
-                  <Route path="/about" element={<About />} />
-                  <Route path="/contact" element={<Contact />} />
-                  <Route path="/" element={<Home />} />
-                </Routes>
-                <LoginDialog />
-                <Footer />
-              </Router>
-            </ScholarshipsProvider>
-          </AuthProvider>
-        </ThemeProvider>
-      </StyledEngineProvider>
-    </FirebaseProvider>
+    <div className="page-container">
+      <FirebaseProvider>
+        <StyledEngineProvider injectFirst>
+          <ThemeProvider theme={theme}>
+            <CssBaseline />
+            <Helmet
+              titleTemplate={`%s | ${BRAND_NAME}`}
+              defaultTitle={
+                BRAND_NAME + ' | Scholarships for Undocumented Students'
+              }
+            />
+            <AuthProvider>
+              <ScholarshipsProvider>
+                <Router>
+                  <div className="content-wrap">
+                    <Header />
+                    <HeaderSkeleton />
+                    <Routes>
+                      <Route
+                        path="/scholarships/new"
+                        element={
+                          <ProtectedRoute element={<AddScholarship />} />
+                        }
+                      />
+                      <Route
+                        path="/scholarships/:id/edit"
+                        element={
+                          <ProtectedRoute element={<EditScholarship />} />
+                        }
+                      />
+                      <Route
+                        path="/scholarships/:id"
+                        element={<ViewScholarship />}
+                      />
+                      <Route
+                        path="/scholarships"
+                        element={<ListScholarships />}
+                      />
+                      <Route path="/about" element={<About />} />
+                      <Route path="/contact" element={<Contact />} />
+                      <Route path="/" element={<Home />} />
+                    </Routes>
+                    <LoginDialog />
+                  </div>
+                  <Footer />
```

Comment:
You need to add padding to the bottom of the side bar otherwise filters can't be applied if the side bar's height exceeds the page's height:
<img width="395" alt="Screen Shot 2022-02-26 at 10 36 10 PM" src="https://user-images.githubusercontent.com/8023233/155871357-07e9299d-ca19-4e4b-a3bb-2c1cf9b7626c.png">

Honestly I would try to use refs instead (I should probably do the same for the app bar), and have the side bar's height reduced by the visible height of the footer.